proto/proto/store/rpc.proto

```diff
@@ -138,6 +138,17 @@ message AccountProofs {
     message AccountProof {
         // State header for public accounts.
         message AccountStateHeader {
+            // Represents a single storage slot with the requested keys and their respective values.
+            message StorageSlotMap {
+                // The storage slot index ([0..255]).
+                uint32 storage_slot = 1;
+
+                // Merkle proofs of the map value as partial sparse merkle tree for compression.
+                // The respective rust types is `SparseMerkleTree` and the transformation to and from
+                // bytes is done via the traits `Serializable::to_bytes` and `Deserializable::from_bytes`.
+                bytes partial_smt = 2;
```

Comment:
I think we are intending to deserialize this as `PartialStorageMap` (cc @igamigo or @SantiagoPittella to confirm). If so, I would name this message `PartialStorageMap` as well.

Also, we've recently modified the definition of `PartialStorageMap` to also include original keys (see https://github.com/0xMiden/miden-base/pull/1878) - so, serializing `PartialSmt` may no-longer be sufficient. So, maybe, what we should do here is serialize `PartialStorageMap`.